extensions/rocksdb-repos/database/RocksDatabase.h

```diff
@@ -59,8 +59,6 @@ class RocksDatabase {
  private:
   const std::string column_;
   std::shared_ptr<RocksDbInstance> db_;
-
-  static std::shared_ptr<core::logging::Logger> logger_;
```

Comment:
what made this necessary? I support the refactor, I'm just interested in the reason, it's something we might need to take into account during code design in the future.

Reply:
When LTO is enabled the order of initialization of the static object changes and the logger_ was not available when it was first used, so it crashed.